tensorflow/compiler/aot/codegen.cc

```diff
@@ -274,7 +274,7 @@ Status GenResultMethods(const tf2xla::Config& config,
 Status GenVariableMethods(const tf2xla::Config& config,
                           const xla::ProgramShapeProto& ps, string* methods) {
   size_t num_args = ps.parameters_size();
-  for (int i = config.feed_size(); i < num_args; ++i) {
+  for (int i = config.feed_size(), end = num_args; i < end; ++i) {
```

Comment:
Why not change type of `num_args` just above?

Reply:
suggested change applied.